include/mgp.py

```diff
@@ -1610,4 +1610,28 @@ def defined_in_this_module(obj: object):
             setattr(module, name, wrap_function(obj))
 
 
+class Logger:
+    """Represents a Logger."""
+
+    __slots__ = ("_logger",)
+
+    def __init__(self):
+        self._logger = _mgp._LOGGER
+
+    def info(self, out: str) -> None:
+        self._logger.info(out)
+
+    def warning(self, out: str) -> None:
+        self._logger.warning(out)
+
+    def critical(self, out: str) -> None:
+        self._logger.critical(out)
+
+    def error(self, out: str) -> None:
+        self._logger.error(out)
+
+    def trace(self, out: str) -> None:
+        self._logger.trace(out)
```

Comment:
I would also opt for using it statically rather than defining a Logger object. The reason is that there is no difference in logging functionality, and it is all going through a single logging source in Memgraph.

Reply:
The reason behind this approach was to make logging the same experience as with the python [`logging` ](https://docs.python.org/3/library/logging.html#logging.Logger)module, where the user can create a logger and set the global level of logger  (we can add this functionality), and  log with just `logger.log("message")`

Reply:
Furthermore, I know now it is not yet implemented, but having different  loggers for different procedures, we could add something like the following:
`[datetime][log_level][procedure_name] <message>`
I as a user would really love this, I know this is not yet implemented, but we can then easily add that part later on when we have different loggers for different procedures.

Reply:
If you have plans for the future, then I am okay to go with it!